### ISSUE TITLE
[Bug]: Focus border is not displayed around item when clicking onto menu button with one single item

### ISSUE THREAD

**Issue by vinguyen12** [microsoft/fluentui#34373]
### Component

MenuButton

### Package version

9.61.6

### React version

18.3.1

### Environment

```shell
System:
    OS: Windows 11 10.0.26100
    CPU: (16) x64 AMD EPYC 7763 64-Core Processor
    Memory: 35.50 GB / 63.95 GB
  Browsers:
    Edge: Chromium (134.0.3124.66)
```

### Current Behavior

No focus border around item when clicking onto menu button with one single item

### Expected Behavior

There should be focus border around the menu item

### Reproduction

https://stackblitz.com/edit/4akpjnzm?file=src%2Fexample.tsx

### Steps to reproduce

1. Click onto menu button to expand the dropdown
2. Observe

Repro video
![Image](https://github.com/user-attachments/assets/4eec6abe-34b7-4938-bd75-7e8f3891d672)

### Are you reporting an Accessibility issue?

yes

### Suggested severity

Low - Has or doesn't need a workaround

### Products/sites affected

OWA

### Are you willing to submit a PR to fix?

no

### Validations

- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [x] The provided reproduction is a minimal reproducible example of the bug.

**Comment by bsunderhus** [comment]
Is this really a bug?! I don't see any issues here. 

The focus border is not showing up because there's no navigation. The previous navigation that opened the menu itself was a mouse navigation (which does not trigger focus border to be visible), so:

1. mouse navigation (no focus border)
2. keyboard navigation that goes to nowhere because there are no other items to be focused (no focus border as there's no navigation?!)

@mshoho , @kolaps33 , @ling1726 WDYT?

**Comment by vinguyen12** [comment]
@bsunderhus Thanks for looking into this.

I agree with you it is reasonable that the focus border not showing when using mouse navigation to open the menu, but when users use keyboard navigation after, I think the focus border should appear to show where the focus is, and since there is no other item for it to go to, it should still appear in that one single item.

**Comment by bsunderhus** [comment]
The mechanism we use to decide if focus border is visible relies on navigation happening @vinguyen12, based on the navigation we can assume if its either a mouse interaction or a keyboard one. Since there's no actual navigation happening in this interaction, the mechanism is not being triggered, right @mshoho, @ling1726 ?!

**Comment by miroslavstastny** [comment]
Inconsistent with Browser behavior, Keyboarg is supposed to emit the event -> Fluent UI bug
